docs/python/inference/notebooks/onnxruntime-tvm-tutorial.ipynb

```diff
@@ -302,29 +302,28 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
+     "name": "stderr",
      "output_type": "stream",
      "text": [
-      "STVM ep options:\n",
+      "[13:00:57] /home/agladyshev/workspace/onnxruntime/onnxruntime/core/providers/tvm/tvm_execution_provider.cc:480: TVM EP options:\n",
       "target: llvm -mcpu=skylake-avx512\n",
       "target_host: llvm -mcpu=skylake-avx512\n",
       "opt level: 3\n",
       "freeze weights: 1\n",
       "tuning file path: \n",
-      "tuning type: Ansor\n",
+      "tuning type: AutoTVM\n",
```

Comment:
About that parameter, I had some issues with it. I had some logs I got on a local machine. I used AutoTVM instead of Ansor and got an error I could not correlate to the value of this parameter. Is there a page on TVM documentation we can refer to?

Reply:
TVM has separated documentation for AutoTVM and AutoScheduler (Ansor) using. I've not seen any for issues related to using one instead of another, but I can say that it can occur due to there are different parser for tuning logs from AutoTVM or AutoScheduler. I think we will extend the current documentation by description devoted to using different optimization system but not now. As I know at the moment we have agreement that AutoTVM is supported on ONNX runtime side, therefore it was set as default. Ansor still has some issues and 'tuning type' is developer's key.  @tmoreau89 please confirm or correct me.

Reply:
I think the ideal solution here is to derive from the logs what kind of tuner is in use; Ansor or AutoTVM. It should be doable so that user-induced errors don't occur. I think we can address this in a separate PR.